packages/admin/src-admin/src/dialogs/AdminUpdater.tsx

```diff
@@ -62,6 +63,8 @@ class AdminUpdater extends Component<AdminUpdaterProps, AdminUpdaterState> {
 
     private readonly link: string;
 
+    private oldVersion: string | undefined;
```

Comment:
```suggestion
    private oldVersion?: string;
```

would recommend this style in general.